you-may-like/index.html

```diff
@@ -4,59 +4,73 @@
       <meta charset="UTF-8">
       <meta http-equiv="X-UA-Compatible" content="IE=edge">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
-      <link rel="stylesheet" href="/styles.css">
+      <link rel="stylesheet" href="../styles.css">
       <link rel="stylesheet" href="styles.css">
       <title>Songs like the album.</title>
       <script src="https://kit.fontawesome.com/72f2609661.js" crossorigin="anonymous"></script>
       <link href="music-note.ico" rel="icon" type="image/x-icon" />
    </head>
    <body>
-      <div id="header">
-         <h1 class="heading"> You May Also Like</h1>
-         <div id="switch">
-            <label class="switch">
-            <input type="checkbox">
-            <span class="slider round"></span>
-            </label>
+      <header>
+         <a href="../index.html">
+         <svg width="40" height="40" viewBox="0 0 20 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+           <path class='arrow' d="M0.792893 6.79289C0.402369 7.18342 0.402369 7.81658 0.792893 8.20711L7.15685 14.5711C7.54738 14.9616 8.18054 14.9616 8.57107 14.5711C8.96159 14.1805 8.96159 13.5474 8.57107 13.1569L2.91421 7.5L8.57107 1.84315C8.96159 1.45262 8.96159 0.819456 8.57107 0.428932C8.18054 0.0384079 7.54738 0.0384079 7.15685 0.428932L0.792893 6.79289ZM18.5 8.5C19.0523 8.5 19.5 8.05228 19.5 7.5C19.5 6.94772 19.0523 6.5 18.5 6.5V8.5ZM1.5 8.5H18.5V6.5H1.5V8.5Z" fill="#496C42"/>
+           </svg>
+         </a>   
+
+         <h1 class="h1">You May Also Like</h1>
+               
+               <!-- Toggler Outside Menu -->
+               <div id="header">
+                 <div id="switch">
+                   <label class="switch">
+                     <input style="height: 0px; width: 0px;" type="checkbox" />
```

Comment:
Can you add the styling to the css, please?

Reply:
done!